tool/tsh/tshconfig.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gravitational/teleport/api/profile"
+	"github.com/gravitational/trace"
+	"gopkg.in/yaml.v2"
+)
+
+// .tsh config must go in a subdir as all .yaml files in .tsh get
+// parsed automatically by the profile loader and results in yaml
+// unmarshal errors.
```

Comment:
Can the profile loader be updated to ignore `config.yaml` file instead?

Reply:
It could be but I think this would mean clusters couldn't be named `config` which doesn't seem ideal

Reply:
This makes sense. Let me check with the product team on this. I think another option could be to just do something like `~/.tshrc` which seems consistent with where e.g. shells keep their preferences and it would have another benefit that we won't have to touch `~/.tsh` directory. But let's check with product first.

Reply:
@lxea We have decided that keeping it under `~/.tsh/config/config.yaml` is ok for now. Let's merge this as-is.